projects/packages/autoloader/changelog/fix-bump_composer_versions_round2

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Updated dependencies.
```

Comment:
More something like this IMO
```suggestion
Significance: major
Type: removed

Drop support for Composer <2.2.
```
Although the "major" might be argued with in the same way people sometimes argue that since Composer handles PHP version compatibility then semver doesn't have to be a major bump.